tests/test_headers.py

```diff
@@ -363,6 +363,25 @@ def test_expires_header(self, client):
 
         assert result.headers['Expires'] == 'Tue, 01 Jan 2013 10:30:30 GMT'
 
+    def test_get_header_as_int(self, client):
+        resource = testing.SimpleTestResource(body=SAMPLE_BODY)
+        client.app.add_route('/', resource)
+        request_headers = {
+            'X-Int-Val': '42',
+            'X-Str-Val': 'test-val',
```

Comment:
Nitpick: For good measure also test that it fails with a fractional number, e.g. `'3.14'`